### ISSUE TITLE
Some DeviantArt images redirect to lower-quality version

### ISSUE THREAD

**Issue by nywilds** [qsniyg/maxurl#92]
I've noticed that for some images on DeviantArt, the "max" image found for it is lower quality. For example, the image here https://www.deviantart.com/neverdying/art/Universe-Effects-Desolation-761880938 is normal on its webpage, but the image it redirects to is noticeably pixelated (both images have the same dimensions).

Info: I am using the Firefox extension.

**Comment by qsniyg** [comment]
Odd, I can't reproduce this... could you link the pixelated version you get from the script (and if possible, also the normal image from the website)?

This is the version I get when I click the image on DeviantArt (zooming it in), then right clicking, Copy image address (i.e. not modified by this script): https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/72c63686-7db6-4280-98e4-55b2c708f401/dcllr3e-106d0ad9-df1f-42e0-a54f-e80b55ce4cb7.jpg?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7InBhdGgiOiJcL2ZcLzcyYzYzNjg2LTdkYjYtNDI4MC05OGU0LTU1YjJjNzA4ZjQwMVwvZGNsbHIzZS0xMDZkMGFkOS1kZjFmLTQyZTAtYTU0Zi1lODBiNTVjZTRjYjcuanBnIn1dXSwiYXVkIjpbInVybjpzZXJ2aWNlOmZpbGUuZG93bmxvYWQiXX0.nQ_YnSqtgHQXGJ9xJT6b2lShzKYj3ggark7qIvvuWkQ

The userscript does not modify the location for me if I visit that page, but if I visit the unzoomed image, it will redirect to the same URL as above (tested via string comparison).

**Comment by nywilds** [comment]
The URL it redirects to is https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/intermediary/f/72c63686-7db6-4280-98e4-55b2c708f401/dcllr3e-106d0ad9-df1f-42e0-a54f-e80b55ce4cb7.jpg (the normal one is the link you posted). However, I just tried installing the extension in a fresh profile and it didn't do the redirect, so there must be some other extension I have installed that is causing this somehow. I guess I should have tried a fresh profile first - sorry for the bother!
